File: PlayMinds-Back/src/main/java/com/coderhouse/pmb/Entitys/Assistant/QuizGame.java
package com.coderhouse.pmb.Entitys.Assistant;

import com.coderhouse.pmb.Entitys.Game;

import java.io.Serializable;
import java.util.List;

public class QuizGame implements Serializable {

    private Game game;
    private List<QuestionOBJ> questions;

    public QuizGame() {
    }
    public QuizGame(Game game, List<QuestionOBJ> questions) {
        this.game = game;
        this.questions = questions;
    }

    public Game getGame() {
        return game;
    }

    public void setGame(Game game) {
        this.game = game;
    }

    public List<QuestionOBJ> getQuestions() {
        return questions;
    }

    public void setQuestions(List<QuestionOBJ> questions) {
        this.questions = questions;
    }
}

File: ExamplesOBJ/models/Entitys/Assistant/QuizGame.ts
import { Game } from "../Game";
import { QuestionOBJ } from "./QuestionOBJ";

export class QuizGame {
  game: Game;
  questions: QuestionOBJ[];

  constructor(game: Game, questions: QuestionOBJ[]) {
    this.game = game;
    this.questions = questions;
  }
}
